### DISCUSSION TITLE
Sconto Percentuale su creazione fattura da CallApi

### DISCUSSION THREAD

**Discussion by SupervisoreOP** [fattureincloud/api#229] [Q&A] [answered]
Buongiorno,

chiedo un supporto perché ho necessità di scontare (in percentuale) il totale della fattura che vado a creare tramite call API V2.
Quindi non di scontare il singolo prodotto.

Tramite piattaforma entrando nella fattura c'è il pulsante "Spesa/Sconto Percentuale" che permette di fare ciò (inserendo esempio il 20% di sconto, viene scontato il totale. (immagine esempio: [https://ibb.co/W5HKWnr](url) ) - Risultato: [https://ibb.co/W5HKWnr](url)

Quindi tramite quale funzione nella call create document è possibile immettere lo stesso sconto?
Ho provato con "amount_due_discount" ed inserendo il valore "20".

Ma nulla il totale è lo stesso e non capisco il funzionamento.

C'è modo di immettere lo sconto totale (in percentuale) e la dicitura personalizzata, tramite la call di creazione fattura?

**Comment by SupervisoreOP** [comment]
Scusate mi sono appena accorto che il link del risultato è errato, fate riferimento a questo: [https://ibb.co/tmx01W1](url)

Grazie.

**Comment by emajo** [comment, marked as answer]
Ciao @SupervisoreOP, il campo `amount_due_discount` è lo sconto/maggiorazione che si trova nella sezione `Opzioni avanzate` da web, ed è espresso in euro non in percentuale, per avere lo sconto percentuale come da web invece deve essere aggiunto un _item_ alla fattura del tipo:
```
{
                "name": "Sconto generale -50%",
                "qty": 1,
                "net_price": -0.21,
                "vat": {
                    "id": 0,
                }
 }
```
il valore da scontare deve essere quindi calcolato dalla vostra applicazione.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @SupervisoreOP la risposta è stata soddisfacente?
